tests/test_pipeline_mixin.py

```diff
@@ -242,12 +240,12 @@ def run_task_tests(self, task, torch_dtype="float32"):
                 commit=commit,
                 torch_dtype=torch_dtype,
             )
+            at_least_one_model_is_tested = True
+
         if task in task_to_pipeline_and_spec_mapping:
             pipeline, hub_spec = task_to_pipeline_and_spec_mapping[task]
             compare_pipeline_args_to_hub_spec(pipeline, hub_spec)
 
-            at_least_one_model_is_tested = True
-
         if not at_least_one_model_is_tested:
```

Comment:
It was a wrong merge with `main`, some tests are marked as skipped while they were not actually skipped